pf/internal/schemashim/block_schema.go

```diff
@@ -95,12 +95,38 @@ func (s *blockSchema) Deprecated() string { return s.block.GetDeprecationMessage
 func (s *blockSchema) MaxItems() int      { return int(s.block.GetMaxItems()) }
 func (s *blockSchema) MinItems() int      { return int(s.block.GetMinItems()) }
 
-func (*blockSchema) ConflictsWith() []string                            { panic("TODO") }
-func (*blockSchema) Default() interface{}                               { panic("TODO") }
-func (*blockSchema) DefaultFunc() shim.SchemaDefaultFunc                { panic("TODO") }
-func (*blockSchema) DefaultValue() (interface{}, error)                 { panic("TODO") }
-func (*blockSchema) ExactlyOneOf() []string                             { panic("TODO") }
-func (*blockSchema) SetElement(config interface{}) (interface{}, error) { panic("TODO") }
-func (*blockSchema) SetHash(v interface{}) int                          { panic("TODO") }
-func (*blockSchema) StateFunc() shim.SchemaStateFunc                    { panic("TODO") }
-func (*blockSchema) UnknownValue() interface{}                          { panic("TODO") }
+func (*blockSchema) ConflictsWith() []string {
```

Comment:
It's probably worth embedding a `schemaGenerationOnly` type in `blockSchema`, `objectPseudoResource` and `typeSchema`. That way we don't need to repeat this message for each type.

Reply:
It's in the "schemashim" package but feel free to rename. I'll just merge for now.